Apps/Src/Temperature.c

```diff
@@ -168,6 +174,8 @@ ErrorStatus Temperature_UpdateSingleChannel(uint8_t channel){
 	if (ERROR == Temperature_ChannelConfig(channel)) {
 		return ERROR;
 	}
+
+	// wait for channel to stabilize
```

Comment:
Was the temperature delay function supposed to be here? If we don't need it anymore we can remove these comments.

Reply:
I'll check if it's used anywhere and remove it if it's not